carapace-server/src/main/java/org/carapaceproxy/core/RuntimeServerConfiguration.java

```diff
@@ -77,7 +77,10 @@ public class RuntimeServerConfiguration {
     private boolean cacheDisabledForSecureRequestsWithoutPublic = false;
     private String mapperClassname;
     private String accessLogPath = "access.log";
+
     private String accessLogTimestampFormat = "yyyy-MM-dd HH:mm:ss.SSS";
+    private SimpleDateFormat accessLogTimestampFormatter;
```

Comment:
I think we should use  DateTimeFormatter, this code is executed by multiple threads since it's called by the handler of the HttpServer.
The differences in the patterns are minimal and for edge cases imho.